client/imports/labpage/Lab

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#header {
+    background-color:black;
+    color:white;
+    text-align:center;
+    padding:5px;
+}
+#terminal{
+    width:350px;
+    float:left;
+    padding:70px;
+    border-left:5px solid #38546d; 
+
+
+}
+#nav {
+    line-height:30px;
+    background-color:#eeeeee;
+    height:300px;
+    width:100px;
+    float:left;
+    padding:5px;
+
+}
+#section {
+    width:350px;
+    float:left;
+    padding:10px;
+}
+#footer {
+    background-color:black;
+    color:white;
+    clear:both;
+    text-align:center;
+    padding:5px;
+}
+</style>
+</head>
+<body>
+
+<div id="header">
+<h1>15131:Great Practical Ideas in CS</h1>
+</div>
+
+<div id="nav">
+Task1<br>
+Task2<br>
+Task3
+</div>
+
+
+
+<div id="section">
+<h2>Lab1: Terminal Usage</h2>
+<p>How to use a command line, how to navigate the filesystem, how to run commands, and more.
+
+</p>
+</div>
+
+
+<div id="terminal">
```

Comment:
I am not sure how this is to be styled to resemble the UX flow- you should be using Material Columns here.